content/en/platform/corda/5.2/developing-applications/cordapp-template/prerequisites/_index.md

```diff
@@ -29,9 +30,9 @@ The {{< version >}} CSDE has been tested with the following:
 | Docker                                   | Docker Engine ~v20.x.y or Docker Desktop ~v3.5.x                    |
 | {{< tooltip >}}Corda CLI{{< /tooltip >}} |                                                                     |
 
-## Required CSDE Ports
+## Required CorDapp Template Ports
 
-The current version of the [Corda combined worker]({{< relref "../installing/_index.md#gradle-helpers-for-the-combined-worker" >}}) runs on ports 5005, 5432, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 5005 for the system Control Center. The following section describes how to make the port available.
+The current version of the [Corda combined worker]({{< relref "../overview/_index.md#gradle-helpers-for-the-combined-worker" >}}) runs on ports 5005, 5432, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 5005 for the system Control Center. The following section describes how to make that port available.
```

Comment:
```suggestion
The current version of the [Corda combined worker]({{< relref "../overview/_index.md#gradle-helpers-for-the-combined-worker" >}}) runs on ports 5005, 7004, and 8888. Reserve these ports to avoid errors. For example, some Mac operating systems use port 5005 for the system Control Center. The following section describes how to make that port available.
```